	I don't know why this is needed. The pdfTeX svn log says
	"Import libpng 1.2.24 into stable". ???

	from:
		Han The Thanh <thanh@pdftex.org>
		Martin Schroeder <martin@pdftex.org>

diff -ur -N xpdf-3.02.orig/xpdf/Decrypt.h xpdf-3.02/xpdf/Decrypt.h
--- xpdf-3.02.orig/xpdf/Decrypt.h	2007-02-27 23:05:52.000000000 +0100
+++ xpdf-3.02/xpdf/Decrypt.h	2008-05-08 15:40:41.000000000 +0200
@@ -5,6 +5,9 @@
 // Copyright 1996-2003 Glyph & Cog, LLC
 //
 //========================================================================
+//  Modified for TeX Live by Peter Breitenlohner <tex-live@tug.org>
+//  See top-level ChangeLog for a list of all modifications
+//========================================================================
 
 #ifndef DECRYPT_H
 #define DECRYPT_H
@@ -73,7 +76,7 @@
 		CryptAlgorithm algoA, int keyLength,
 		int objNum, int objGen);
   virtual ~DecryptStream();
-  virtual StreamKind getKind() { return strWeird; }
+  virtual StreamKind getKind() { return strCrypt; }
   virtual void reset();
   virtual int getChar();
   virtual int lookChar();
diff -ur -N xpdf-3.02.orig/xpdf/Stream.h xpdf-3.02/xpdf/Stream.h
--- xpdf-3.02.orig/xpdf/Stream.h	2007-11-12 13:27:21.000000000 +0100
+++ xpdf-3.02/xpdf/Stream.h	2008-05-08 15:40:41.000000000 +0200
@@ -5,6 +5,9 @@
 // Copyright 1996-2003 Glyph & Cog, LLC
 //
 //========================================================================
+//  Modified for TeX Live by Peter Breitenlohner <tex-live@tug.org>
+//  See top-level ChangeLog for a list of all modifications
+//========================================================================
 
 #ifndef STREAM_H
 #define STREAM_H
@@ -34,6 +37,7 @@
   strFlate,
   strJBIG2,
   strJPX,
+  strCrypt,
   strWeird			// internal-use stream types
 };
 
